script/setup_docker_prereqs

```diff
@@ -1,63 +1,120 @@
 #!/bin/bash
 # Install requirements and build dependencies for Home Assinstant in Docker.
 
+INSTALL_TELLSTICK="${INSTALL_TELLSTICK:-yes}"
+INSTALL_OPENALPR="${INSTALL_OPENALPR:-yes}"
+INSTALL_FFMPEG="${INSTALL_FFMPEG:-yes}"
+INSTALL_PHANTOMJS="${INSTALL_PHANTOMJS:-yes}"
+
 # Required debian packages for running hass or components
 PACKAGES=(
+  # build-essential is required for python pillow module on non-x86_64 arch
+  build-essential
+  # libxrandr-dev is required for openalpr
+  libxrandr-dev
```

Comment:
This package is required to stay installed for OpenALPR, otherwise `apt-get autoremove` removes `libopencv-dev` package.